app/src/main/java/com/github/multimatum_team/multimatum/DeadlineNotification.kt

```diff
@@ -36,16 +33,18 @@ class DeadlineNotification {
         Set a notification that will be triggered in a given time in ms.
         you can pass a title/description and Id in parameter
         */
-        fun setNotification(timeMS: Long, title: String, description: String, id: Int, context: Context){
+        fun setNotification(deadline: Deadline, context: Context){
             alarmManager = context.getSystemService(AppCompatActivity.ALARM_SERVICE) as AlarmManager  //this get an service instance of AlarmManager
             val intent = Intent(context, ReminderBroadcastReceiver::class.java) //this create an intent of broadcast receiver
             //Adding extra parameter that will be used in the broadcase receiver to create the notification
-            intent.putExtra("title", title)
-            intent.putExtra("description", description)
-            intent.putExtra("id", id)
+            intent.putExtra("title", deadline.title)
+            intent.putExtra("description", deadline.description)
+            intent.putExtra("id", deadline.id)
 
             //set the receiver as pending intent
-            pendingIntent = PendingIntent.getBroadcast(context, id, intent, PendingIntent.FLAG_IMMUTABLE)
+            pendingIntent = PendingIntent.getBroadcast(context, deadline.id, intent, PendingIntent.FLAG_IMMUTABLE)
+
+            val timeMS: Long = 0 //get milis from deadline.date (LocalDate)
```

Comment:
What is this line doing? I don't understand the comment.

Reply:
This is creates a pending intent of broadcastreceiver that’ll be trigger by the alarm.
But that was already part of the code of last week.

Reply:
I am asking now because I didn't review this last week :) . But I was asking about `val timeMS: Long = 0`. I don't understand what the comment means, and what timeMs is.

Reply:
Yeah, it's because timeMs is the time when the notification will be set. So I have to get it from notification date, however we haven't decide yet how it'll be manage (when exactly the notif have to be launch). So I just set it to 0 for now (I think I'll change to currentTime.getMilis in order to make it testable)

Reply:
OK, thanks for the explanation. Maybe you can put a comment like "Temporary"?